src/argilla/server/models/mixins.py

```diff
@@ -158,6 +159,10 @@ async def save(self, db: "AsyncSession", autocommit: bool = True) -> Self:
         return self
 
 
+def _default_inserted_at(context: DefaultExecutionContext) -> datetime:
+    return context.get_current_parameters()["inserted_at"]
```

Comment:
To make it work in all the cases:

```suggestion
    return context.get_current_parameters(isolate_multiinsert_groups=False)["inserted_at"]
```

Reply:
Thanks, I saw this this morning, but I don't really understand why this solves the tests. Related to async connections?